app/helpers/SessionAnswersHelper.scala

```diff
@@ -262,4 +264,28 @@ class SessionAnswersHelper @Inject()() extends ImplicitDateFormatter {
       )
     )
   }
+
+  def getPreviousUploadsFileNames()(implicit request: UserRequest[_]): Future[String]= {
+    for {
+      previousUploads <- uploadJourneyRepository.getUploadsForJourney(request.session.get(SessionKeys.journeyId))
+    } yield {
+      val previousUploadsFileName = previousUploads.map(_.map(file => file.uploadDetails.map(details => details.fileName)))
+      previousUploadsFileName.getOrElse(Seq.empty).collect {
+        case Some(x) => x
+      }.mkString(", ")
+    }
+  }
+
+  def getContentWithExistingUploadFileNames(reasonableExcuse : Option[String] = None)(implicit request: UserRequest[_], messages: Messages): Future[Seq[(String, String, String)]]={
+    if(!reasonableExcuse.equals(Some("other"))){
```

Comment:
If you remove the option you can just do 
```suggestion
    if(!reasonableExcuse.equals("other")){
```